src/app/components/profesionales/listar-profesionales.html

```diff
@@ -1,281 +1,210 @@
+
+                        <plex-badge class="mr-1" type="info" tooltip="Año de gracia"
+                                    *ngIf="profesional.formacionPosgrado?.length && contarTiposDePosgrados(indiceProfesional, 'anioDeGracia')">
+                            {{contarTiposDePosgrados(indiceProfesional, 'anioDeGracia')}}
+                        </plex-badge>
+                        <plex-badge class="mr-1" type="danger" tooltip="Vencidas"
+                                    *ngIf="profesional.formacionPosgrado?.length && contarTiposDePosgrados(indiceProfesional, 'vencida')">
+                            {{contarTiposDePosgrados(indiceProfesional, 'vencida')}}
+                        </plex-badge>
+                        <plex-badge class="mr-1" type="warning" tooltip="Suspendida"
+                                    *ngIf="profesional.formacionPosgrado?.length && contarTiposDePosgrados(indiceProfesional, 'suspendida')">
+                            {{contarTiposDePosgrados(indiceProfesional, 'suspendida')}}
+                        </plex-badge>
+
+                    </td>
+
+                    <td class="column-right my-3 d-flex align-items-center">
+                        <plex-button type="info" icon="eye" size="sm" (click)="showProfesional(profesional)"
+                                     tooltip="Ver profesional">
+                        </plex-button>
+                        <plex-dropdown type="link" right="true" size="sm" icon="dots-vertical" [items]="itemsDropdown"
+                                       (onOpen)="setDropDown(profesional)"></plex-dropdown>
+                    </td>
+                </tr>
+            </plex-table>
+        </div>
+
+        <div *ngIf="nuevoProfesional">
+            <plex-title titulo="Nuevo profesional" size="lg" *ngIf="nuevoProfesional">
+                <plex-button *ngIf="nuevoProfesional" label="Volver" type="danger" (click)="verNuevoProfesional(false)">
+                </plex-button>
+            </plex-title>
+            <app-profesional [confirmar]='confirmar' [nuevoProf]='true' [editable]='false'></app-profesional>
+        </div>
+        <div *ngIf="editar">
+            <app-profesional [profesional]='profesionalSeleccionado' [showOtraEntidadFormadora]='false'
+                             [confirmar]='confirmar' (editado)="editar=false" [desdeListaProfesionales]="true"
+                             [editable]='true'>
+            </app-profesional>
+        </div>
+    </plex-layout-main>
+
+    <plex-layout-sidebar type="invert">
+        <div *ngIf='seleccionado'>
+            <app-listar-profesionales-detalle [profesionalSeleccionado]="profesionalSeleccionado"
+                                              (cancelar)="cancelarDetalle($event)"
+                                              (editarSidebar)="editarProfesional($event)"
+                                              [idProfesional]="profesionalSeleccionado.id">
+            </app-listar-profesionales-detalle>
```

Comment:
La condición podria estar directamente en el app-listar-profesionales-detalle
```suggestion
            <app-listar-profesionales-detalle *ngIf='seleccionado' [profesionalSeleccionado]="profesionalSeleccionado"
                                              (cancelar)="cancelarDetalle($event)"
                                              (editarSidebar)="editarProfesional($event)"
                                              [idProfesional]="profesionalSeleccionado.id">
            </app-listar-profesionales-detalle>
```